update-bot/README.md

```diff
@@ -17,15 +15,10 @@ cp user-config.template.py user-config.py
 ```
 cp local_family.template.py local_family.py
 ```
-4. Clone the pywikibot repository:
-   `git clone --branch 2.0 --recursive  https://gerrit.wikimedia.org/r/pywikibot/core.git ~/src/pywikibot`
-5. Install the necessary libraries:
+4. Install the necessary libraries:
    `pip install -r update-bot/requirements.txt`
-6. Set and export the library path:
-```
-PYTHONPATH=$PYTHONPATH:~/src/pywikibot
-export PYTHONPATH
-```
+5. To run tests install dev libraries (optional)
+   `pip install -r update-bot/dev-requirements.txt`
```

Comment:
Really minor thing but maybe could be also considered at that point: this README file is not entirely consistent when it comes to refering to filepath. The line above mentions the `update-bot` directory, same as line 19. On the contrary, steps on creating config files (e.g. lines 12 and 16) do not mention the directory, same in "General usage information" section.
As this README file is actually in the `update-bot`, maybe the directory could be left over?